### PR TITLE
enable arm32/arm64 target for .net apps built against OnnxRuntime.ML.OnnxRuntime

### PR DESCRIPTION
**Pull request by jywu-msft**
couldn't build arm64 .net app due to target file not allowing it.

### CODE REVIEW

**Review by pranavsharma — APPROVED** [review]

**Comment by pranavsharma** [comment]
@snnn do we've arm64 machines in CI now so that such issues can be validated?

File: csharp/src/Microsoft.ML.OnnxRuntime/targets/netstandard/targets.xml [hunk 1]
```diff
@@ -6,12 +6,12 @@
     the PlatformTarget is empty, and you don't know until runtime (i.e. which dotnet.exe)
     what processor architecture will be used.
   -->
-    <Error Condition="('$(PlatformTarget)' != 'x64' AND '$(PlatformTarget)' != 'x86' AND '$(PlatformTarget)' != 'AnyCPU') AND
+    <Error Condition="('$(PlatformTarget)' != 'x64' AND '$(PlatformTarget)' != 'arm64' '$(PlatformTarget)' != 'x86' AND '$(PlatformTarget)' != 'AnyCPU') AND
```
Comment (HectorSVC) [inline csharp/src/Microsoft.ML.OnnxRuntime/targets/netstandard/targets.xml]: arm64' '$(PlatformTarget)' => arm64' AND '$(PlatformTarget)'

Comment (jywu-msft) [reply csharp/src/Microsoft.ML.OnnxRuntime/targets/netstandard/targets.xml]: whoops. fixed.

**Comment by HectorSVC** [comment]
Should we also add arm32 since there's win-arm binaries also in Ort native nuget package?

**Review by HectorSVC — APPROVED** [review]

**Review by snnn — APPROVED** [review]

**Comment by snnn** [comment]
> @snnn do we've arm64 machines in CI now so that such issues can be validated?

We don't build .net packages in such a way. This PR is only needed for who want a packages that only contains ARM binaries.

**Pull request merged**